network-api/networkapi/news/factory.py

```diff
@@ -32,34 +32,15 @@ class Meta:
         exclude = ("headline_sentence",)
 
     class Params:
-        unpublished = Trait(
-            publish_after=Faker("date_time", tzinfo=timezone.utc)
-            if RANDOM_SEED and not TESTING
-            else Faker("future_datetime", end_date="+30d", tzinfo=timezone.utc)
-        )
-        has_expiry = Trait(
-            expires=Faker("date_time", tzinfo=timezone.utc)
-            if RANDOM_SEED and not TESTING
-            else Faker("future_datetime", end_date="+30d", tzinfo=timezone.utc)
-        )
-        expired = Trait(
-            expires=Faker("date_time", tzinfo=timezone.utc)
-            if RANDOM_SEED and not TESTING
-            else Faker("past_datetime", start_date="-30d", tzinfo=timezone.utc)
-        )
         video = Trait(is_video=True)
 
     headline = LazyAttribute(lambda o: o.headline_sentence.rstrip("."))
     outlet = Faker("company")
-    date = Faker("date") if RANDOM_SEED and not TESTING else Faker("past_date", start_date="-30d")
+    date = Faker("past_date", start_date=datetime(2020, 1, 1), tzinfo=timezone.utc)
```

Comment:
I'm unsure why we needed to have two ways to generate values for `date` here. Any insights?